### DISCUSSION TITLE
Thảo luận tuần 4

### DISCUSSION THREAD

**Discussion by GiaHan06** [GiaHan06/Lop-10A02-Nhom-02#20] [Ideas]
Zô coi tui phân công nhiệm vụ nè

**Comment by GiaHan06** [comment]
về cái phần nội dung thì tui nói lại nha
mình sẽ nói về những khó khăn trong đại dịch covid, phân tích những ý nhỏ của nó ra. ví dụ như ảnh hưởng đến nền kinh tế nói nó ảnh hưởng ra sao, gây ra những hậu quả gì...; sau đó là từ những cái khó khăn đó thì ta thấy được tinh thần đoàn kết và tình thương người của người dân, giúp đỡ nhau vượt qua đại dịch các thứ

**Comment by GiaHan06** [comment]
_Về phần khó khăn_ @chaudambaotram tìm thêm ý nha, t sẽ tìm phụ m luôn
_Về phần tình người_ thì lúc lần trước Qanh đã tìm đươc cũng kha khá ý nên là @LTGHan m xem còn thiếu gì nữa hong thì bổ sung nữa nha
_Về phần đoàn kết_ @HuynhLeBaoTran bà tìm ý nha bà
@NguyenNgocQuynhAnh thì tìm thơ ca dao tục ngữ về tình đoàn kết nha m, tìm 2-3 cái để xem cái nào ok nhất nha
_Mấy bà cứ tìm thông tin nha t sẽ hỗ trợ hết nên đừng lo, nhưng mà nhớ làm_

**Reply by chaudambaotram** [reply]
ukii

**Reply by NguyenNgocQuynhAnh** [reply]
kê nka

**Comment by LTGHan** [comment]
Phần tình người á thì mình sẽ thêm thắt dẫn chứng giống như này he https://kiengiang.gov.vn/m/263/28820/Tinh-nguoi-trong-phong--chong-dich-COVID-19.html

**Reply by GiaHan06** [reply]
Lưu Hân mãi đỉnk mãi mận cóc ổi xoài vải :>
